packages/example/src/index.tsx

```diff
@@ -41,6 +46,8 @@ const PERSISTENCE_KEY = 'NAVIGATION_STATE';
 Asset.loadAsync(StackAssets);
 
 export default function App() {
+  const containerRef = React.useRef<NavigationContainerRef>(React.createRef());
```

Comment:
You don't need to pass `createRef` into `useRef`, we can pass null and use the returned ref instead of `.current`